### PR TITLE
Commerce 8388 3/3 checkout

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-8388

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-channel-web/src/main/java/com/liferay/commerce/channel/web/internal/servlet/taglib/DefaultCommercePaymentMethodsShippingOptionsJSPDynamicInclude.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
+/**
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-channel-web/src/main/java/com/liferay/commerce/channel/web/internal/servlet/taglib/DefaultCommercePaymentMethodsShippingOptionsJSPDynamicInclude.java:1]: move to 2nd pr

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
Ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:forward

**Pull request closed without merging**